src/copycutter/text_question.py

```diff
@@ -30,16 +30,12 @@ class Backend(enum.Enum):
     cookie = enum.auto()
     copier = enum.auto()
 
+
 NAMES = ["Form", "Code-Browser"]
 
 
 class TextQuestion(Static):
-    _label: Label
-    _input: Input
-    _property_val: str
-    _prompt: str
-
-    def __init__(self, placeholder, property_val, prompt, **kwargs):
+    def __init__(self, placeholder: str, property_val: str, prompt: str, **kwargs: Any):
```

Comment:
Unless you are using dataclasses, these are picked up from being set inside init.